### ISSUE TITLE
corrupt wav file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bleskodev/rubyripper#468]
```
1) Please describe the steps to reproduce the situation:
a. on debian squeeze rip a disc to a single wav file with a cue sheet via cli
b. move that wav to a windows xp machine and try to play it via foobar2k or 
windows media player
c.

2) What is the expected output? What do you see instead?

I expect the sweet tones of awesome music but i get an error that says the file 
is an unsupported format or corrupt

3) What version of rubyripper are you using? On what operating system? The
gtk2 of commandline interface?

I'm using the latest git repo via cli

4) Is this not already fixed with the latest & greatest code? See for
instructions the Source tab above.

apparently not

5) Does the problem happen with all discs? If not, please attach
the output of cdparanoia -Q with a disc that gives trouble.

All discs

6) Please explain why this change is important for you. Also, how many
users would benefit from this change?

Basically I'm trying to confirm that ruby ripper compares to eac, but i have 
yet to create a wav file that actually plays anywhere but the host system.

Please provide any additional information below. The more usefull
information provided, the sooner the issue will be fixed.

Thinking that something was goofy with the wav file, I tried going straight to 
flac, but the flac encoder threw the same error, that it couldn't encode the 
file because it was corrupted


```

Original issue reported on code.google.com by `hender...@gmail.com` on 21 Jul 2011 at 4:03
- Blocking: #556

Attachments:
- [ripping.log](https://storage.googleapis.com/google-code-attachments/rubyripper/issue-468/comment-0/ripping.log)
